src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,134 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.InvalidDataException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static StorageDao storageDao;
+    private static RegistrationService registrationService;
+    private User user;
+
+    @BeforeAll
```

Comment:
Consider using @BeforeEach instead of @BeforeAll for initializing instances that are used in multiple tests to ensure that tests do not depend on each other. @BeforeAll is used for static initializations and is executed once before all tests, while @BeforeEach is executed before each test, providing a fresh instance for each test case.